pyiron_base/jobs/job/runfunction.py

```diff
@@ -431,6 +444,91 @@ def run_job_with_runmode_srun(job):
     )
 
 
+def run_job_with_runmode_executor(job, executor, gpus_per_slot=None):
```

Comment:
Should this or `run_job_with_runmode_executor_futures` be decorated with `run_time_decorator` or is this somehow taken care of the `multiprocess_wrapper`?